packages/legend-extension-dsl-mastery/src/graph-manager/protocol/pure/v1/transformation/pureProtocol/V1_DSL_Mastery_ProtocolHelper.ts

```diff
@@ -925,3 +926,50 @@ export const V1_masterRecordDefinitionModelSchema = (
       ),
     ),
   });
+
+/**********
+ * runtime
+ **********/
+
+export const V1_serializeRuntime = (
+  protocol: V1_Runtime,
+  plugins: PureProtocolProcessorPlugin[],
+): PlainObject<V1_Runtime> => {
+  const extraRuntimeSerializers = plugins.flatMap(
+    (plugin) =>
+      (
+        plugin as DSL_Mastery_PureProtocolProcessorPlugin_Extension
+      ).V1_getExtraRuntimeProtocolSerializers?.() ?? [],
+  );
+  for (const serializer of extraRuntimeSerializers) {
+    const runtimeJson = serializer(protocol, plugins);
+    if (runtimeJson) {
+      return runtimeJson;
+    }
+  }
+  throw new UnsupportedOperationError(
+    `Can't serialize runtime Json: no compatible serializer available from plugins`,
```

Comment:
Please modify this to `Can't serialize mastery runtime: no compatible serializer available from plugins